lib/katello/tasks/upgrades/3.10/refresh_puppet_distributor_config.rake

```diff
@@ -0,0 +1,17 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.10' do
+      desc "Refresh Puppet Distributor configurations"
+      task :refresh_puppet_distributor_config => %w(environment) do
+        User.current = User.anonymous_admin
+
+        Katello::RootRepository.puppet_type.find_each do |root_repo|
+          root_repo.repositories.each do |repo|
+            puts "Refreshing repository #{repo.label} (#{repo.id})"
+            ForemanTasks.sync_task(::Actions::Pulp::Repository::Refresh, repo)
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
upgrade tasks should be added to seeds.d/111-upgrade_tasks.rb now.  Otherwise this looks good.

Reply:
added